.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,26 @@
+name: Node CI
+
+on: [push]
```

Comment:
Mogłbyś może mi krótko opisać za co jest ta linia odpowiedzialna? Jedyna linia której zupełnie nie rozumiem, a w dokumentacji też nie mogę znaleźć co to robi :)

Reply:
> Runs your workflow when someone pushes to a repository branch, which triggers the push event.
To też była domyślna wartość w takich podstawowym szablonie dla aplikacji NodeJS.
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#push-event-push